provisioning/utils.go

```diff
@@ -104,7 +98,11 @@ func GetIronicIP(client kubernetes.Interface, targetNamespace string, config *me
 	}
 
 	if UseIronicProxy(config) {
-		ironicIP, err = getServerInternalIP(osclient)
+		internalIPs, err1 := getServerInternalIPs(osclient)
+		if err1 != nil {
+			return
```

Comment:
The naked `return` will return whatever is stored in `err` as the error value.

Reply:
@honza should I just print the error using `fmt` or return an error message. And I think I missed the case of `internalIPs=nil`, so will handle it as well.

Reply:
I think you should be able to assign to `err`, and then do a naked return.

```go
internalIPs, err := getServerInternalIPs(osclient)
if err != nil {
    return
}
```

Reply:
@honza does this look okay now ?